Packs/CommunityCommonDashboards/Dashboards/dashboard-9347db72-018a-4b49-8f3d-f11d927720ee-CISOMetrics_copy.json

```diff
@@ -0,0 +1,1499 @@
+{
+    "fromDate": "0001-01-01T00:00:00Z",
+    "fromDateLicense": "0001-01-01T00:00:00Z",
+    "id": "CISOMetrics_copy",
```

Comment:
@rurhrlaub 
Please change the name so it won't contain "_copy". If you copied it from another dashboard, provide a different name.
Also, the name of the file should be as the name of the dashboard with a ".json" extension.

Reply:
DONE